feat(web): Enable selection interactions in folder view

arnolicious opened the pull request:
So I don't know if this was intentionally omitted, or just wasn't implemented yet.

I hope it's the latter, since I would find this very useful, aswell as a few other people in #14307

I haven't been able to test it completely, since I'm on windows and my bind mounts seem to be a lil fucked up, resulting in folders with only one image in it, so some other testers would be greatly appreciated.

If this was indeed intentionally omitted, feel free to close the PR

#### Edit
I also added a button in the Asset View Detail Pane that links to the containing folder. That way, when you open an asset from the timeline or search, you can jump straight into the folder view to the correct folder.

This button only shows up, if folders are enabled and the Folder sidebar link is also activated.

Closes #14307

arnolicious committed:
feat(web): Enable selection interactions in folder view

alextran1502 commented on web/src/routes/(user)/folders/[[photos=photos]]/[[assetId=id]]/+page.svelte:
```diff
@@ -50,8 +65,59 @@
+  const triggerAssetUpdate = () => {};
+
+  const onAddToAlbum = (assetIds: string[]) => {
+    // if (terms.isNotInAlbum.toString() == 'true') {
```
Can we handle this case as well?

alextran1502 commented on web/src/routes/(user)/folders/[[photos=photos]]/[[assetId=id]]/+page.svelte:
```diff
@@ -50,8 +65,59 @@
+  };
+
+  const onAssetDelete = (assetIds: string[]) => {
+    // const assetIdSet = new Set(assetIds);
```
Can we handle this case as well?

arnolicious committed:
feat(web): Add link to parent folder in detail pane, if folders are enabled

arnolicious committed:
Added invalidation and refreshing of asset cache on changes

arnolicious commented:
I implemented the refreshing of assets @alextran1502 

I also added a button in the Asset View Detail Pane that links to the containing folder. That way, when you open an asset from the timeline or search, you can jump straight into the folder view to the correct folder.

This button only shows up, if folders are enabled and the Folder sidebar link is also activated.

alextran1502 commented:
Hi @arnolicious, I just tested the PR, and it works well. 

For navigating to the folder from the detail panel, can we make this link clickable instead?

<img width="359" alt="image" src="https://github.com/user-attachments/assets/ab354caa-4c95-4df0-88e2-6a9ca0157b7e" />

alextran1502 commented on web/src/lib/components/asset-viewer/detail-panel.svelte:
```diff
@@ -31,6 +31,9 @@
     mdiClose,
     mdiEye,
     mdiEyeOff,
+    mdiFolder,
```
Please remove the unused code so the tests pass, thank you

arnolicious committed:
fix: removed unused imports and changed link

alextran1502 committed:
chore: styling

alextran1502 committed:
lint
